doc/src/input/systems/interactions/electrostatic.rst

```diff
@@ -59,6 +59,16 @@ Usually 7-8 is a good value for pure water, for a very periodic charges
 distribution (like a crystal) a lower value, such as 5 is sufficient, and for
 more heterogeneous system, higher values of ``kmax`` are needed.
 
+It is also possible to set the ``alpha`` parameter of Ewald solver directly with
+the corresponding keyword. If no value is provided for alpha, the default is to
+use :math:`\pi / \text{cutoff}`. A good value of ``alpha`` is such as
```

Comment:
Can we rephrase that sentence? I feel like it is unclear to the reader if there is a default used when omitting a value or if one should set the value to be PI/rc.

Reply:
I'd replace "... such as ..." by something like "A good value for ``alpha`` is one that satisfies :math:....

Reply:
Updated! Tell me what you think of the new formulation.